modules/repo/main.tf

```diff
@@ -10,6 +10,12 @@ variable "visibility" {
   default     = "public"
 }
 
+variable "ensure_commitlint" {
+  type = bool
+  description = "Whether or not this repo has the commitlint github action installed."
+  default = "false"
```

Comment:
IIUC, this PR just makes terraform_github have the commitlint action, because this defaults to false, and line 1055 sets it to true.  If we want to force all repos to have it, we would change this line to true, correct?